{
    "data_folder_path": "data/CVE-2016-4009",
    "pull_requests_urls": [
        "https://github.com/python-pillow/Pillow/pull/1714"
    ],
    "cve_name": "CVE-2016-4009",
    "repo": "python-pillow/Pillow",
    "warnings": {
        "cve": "CVE-2016-4009",
        "warnings": [
            {
                "type": "NOT_MODIFIED",
                "file_name": "Tests/test_image_resample.py",
                "message": "not a modified file (status: FileStatus.ADDED)\n"
            }
        ]
    },
    "files_extension_stats": {
        ".c": 2,
        ".py": 1
    },
    "file_number": 3,
    "removed_files_number": 0,
    "added_files_number": 0,
    "nb_lines_affected": 52,
    "private_key_files": [],
    "languages": [
        "C",
        "Python"
    ]
}